modules/op-providing-secure-connection.adoc

```diff
@@ -3,9 +3,9 @@
 
 To create a route with the re-encrypted TLS termination, run:
 
-[source,terminal,subs="atrributes+"]
+[source,terminal,subs="attributes+"]
```

Comment:
@Preeticp there are a handful of pipelines files that I touched in this PR. Can you (or someone else from the team) please ack whether these updates are okay?

Reply:
@Preeticp I'm going to go ahead and merge this, since I'm pretty confident with these changes to the pipelines docs. But please let me know if you do see anything here that isn't right. Thanks!

CC @HarshCasper

Reply:
@bergerhoffer thanks a lot for doing this. All the changes look good to me. I will share this with the team so that we can avoid such mistakes. Sorry about the late reply, was on sick leave on Monday, and there was just too much to catch up with yesterday :(